combustion-android-ble/build.gradle

```diff
@@ -74,7 +74,17 @@ afterEvaluate {
                 // You can then customize attributes of the publication as shown below.
                 groupId = "inc.combustion"
                 artifactId = "framework"
-                version = "0.0.2"
+                version = "0.0.3"
+            }
+            // Creates a Maven publication called "debug".
+            debug(MavenPublication) {
+                // Applies the component for the debug build variant.
+                from components.debug
+
+                // You can then customize attributes of the publication as shown below.
+                groupId = "inc.combustion"
+                artifactId = "framework"
+                version = "0.0.3"
```

Comment:
Not necessary now, but can this be grabbed from a variable so that it doesn't have to be set for each configuration?